packages/react-core/src/components/Slider/Slider.tsx

```diff
@@ -326,7 +326,10 @@ export const Slider: React.FunctionComponent<SliderProps> = ({
       return (
         <InputGroup>
           {textInput}
-          <InputGroupText variant="plain"> {inputLabel}</InputGroupText>
+          <InputGroupText {...(isDisabled && { className: css(styles.modifiers.disabled) })}>
+            {' '}
```

Comment:
Just curious what this is for? I don't think a space would ever actually render unless there were something before it.

Reply:
Ah good catch. It looks like there was a whitespace originally and it got converted to this.